Plate_2_data/4.processing_features/processing_features_plate2.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# initialize the correct shell for your machine to allow conda to work (see README for note on shell names)
+conda init bash
+# activate the main conda environment
+conda activate interstellar_data
+
+# convert all notebooks to python files into the scripts folder
+jupyter nbconvert --to python --output-dir=scripts/ *.ipynb
```

Comment:
I think since this file in the directory with the four notebooks, then I would rather have one line that converts all of the notebook at once. It makes it look cleaner to me but I can see why `*` could not be preferred depending on the situation.